provider/ansible.rb

```diff
@@ -83,7 +83,9 @@ def unicode_string(string)
       end
 
       def build_url(url)
-        "\"#{url.gsub('{{', '{').gsub('}}', '}')}\""
+        # Return a quoted string, with single pairs of {} brackets and all
+        # requested strings are underscored (as they come from the Ansible configs)
+        "\"#{url.gsub(/{{\w+}}/) { |param| "{#{param[2..-3].underscore}}" }}\""
```

Comment:
Why does only part of `param` need to be underscored?

Reply:
I can add a comment for this.

The way Ruby's regex function thing here works is that it grabs every instance of "{{word}}" and runs a function on it.

The function inside the block needs to remove the second set of brackets and underscore the inner word. The way I'm doing it is to slice away the "{{" and "}}" from "{{word}}", underscore the word and add the brackets back in.